controllers/errorHandle.controller.js

```diff
@@ -0,0 +1,8 @@
+exports.RouteNotFoundError = (err, req, res, next) => {
```

Comment:
you have an app.all which is handling the route not found error what is this one doing?

Reply:
Oh yeah, I will combine this with my app.all one and remove